tests/odov3/odo_test.go

```diff
@@ -43,11 +36,12 @@ import (
 )
 
 var stacksDir string
+var filesStr string
 
 func init() {
 	rand.Seed(time.Now().UnixNano())
 	flag.StringVar(&stacksDir, "stacksDir", "../../stacks", "The directory containing the stacks")
-
+	flag.StringVar(&filesStr, "filesStr", "", "The files to test as a string separated by spaces")
```

Comment:
Something like `stacks` or `stackFiles` instead IMO. Not crazy about `filesStr` as an argument

Reply:
Yeah, I wasn't too keen on this as well. How about `stackDirs`?

Reply:
stackDirs is fine